jwst/transforms/models.py

```diff
@@ -593,3 +595,48 @@ def __repr__(self):
         return "{0}(condition={1}, compareto={2}, value={3})".format(self.__class__.__name__,
                                                                      self.condition, self.compareto,
                                                                      self.value)
+
+
+class V23ToSky(Rotation3D):
+
+    inputs = ("v2", "v3")
+    outputs = ("ra", "dec")
+
+    @staticmethod
+    def spherical2cartesian(alpha, delta):
+        """
+        Convert spherical coordinates (in deg) to cartesian.
+        """
+        alpha = np.deg2rad(alpha)
+        delta = np.deg2rad(delta)
+        x = np.cos(alpha) * np.cos(delta)
+        y = np.cos(delta) * np.sin(alpha)
+        z = np.sin(delta)
```

Comment:
Could you confirm these are correct? I was thinking they were x = cos sin, y = sin sin and z = cos (of the angles). It could be some orientation thing or I am not remembering well (more likely the latter).